bpaotu/bpaotu/settings.py

```diff
@@ -13,6 +13,10 @@
 SCRIPT_NAME = env.get("script_name", os.environ.get("HTTP_SCRIPT_NAME", ""))
 FORCE_SCRIPT_NAME = env.get("force_script_name", "") or SCRIPT_NAME or None
 
+# This should be the path under the webapp is installed on the server ex. /bpa/otu on staging
+# TODO I think this is alwasy SCRIPT_NAME if not get separately from enviroment
+BASE_URL = SCRIPT_NAME
+
```

Comment:
Django's handling of this stuff is a bit of pain - I'll see how it goes when I push the release to staging under `/bpa/otu`